### PR TITLE
feat: update Guava and Lombok Version

### PR DESCRIPTION
**Pull request by leechor**

### CODE REVIEW

**Comment by aiwenmo** [comment]
Hi. We have a contributor discussion group on WeChat, inviting you to join the group.

**Comment by leechor** [comment]
> Hi. We have a contributor discussion group on WeChat, inviting you to join the group.

how to join.

**Comment by aiwenmo** [comment]
> > Hi. We have a contributor discussion group on WeChat, inviting you to join the group.
> 
> how to join.

Enter the WeChat user community group (recommended, add WeChat wenmo_ai to invite into the group) and QQ user community group (543709668) to communicate, and apply for the remarks "Dinky + company name + position".

File: pom.xml [hunk 1]
```diff
@@ -60,9 +60,9 @@
         <hutool.version>5.8.4</hutool.version>
         <druid-starter>1.2.8</druid-starter>
         <mybatis-plus-boot-starter.version>3.5.1</mybatis-plus-boot-starter.version>
-        <lombok.version>1.18.16</lombok.version>
+        <lombok.version>1.18.24</lombok.version>
         <jackjson.version>2.12.1</jackjson.version>
-        <guava.version>21.0</guava.version>
+        <guava.version>31.1-jre</guava.version>
         <mysql-connector-java.version>8.0.28</mysql-connector-java.version>
```
Comment (aiwenmo) [inline pom.xml:66]: The upgraded version needs to test the function of task submission.

Comment (leechor) [reply pom.xml:66]: > The upgraded version needs to test the function of task submission.

It's been a good run until now.

Comment (aiwenmo) [reply pom.xml:66]: > It's been a good run until now.

Yes, please wait for me to test it.

**Pull request merged**